docs/PUBLISHING.md

```diff
@@ -121,6 +121,7 @@ Release JSON valid!
    ```
 
    _NOTE: this will be repeated each time you have a new version of your dApp to release. The mint address of the latest release is recorded in your configuration file_.
+   > **Warning**: when doing this step make sure your network is reliable and can handle upload speeds of 25 megabytes within 100 seconds (default chunk size for a default timeout)_.
```

Comment:
This is awesome! Just a couple of requested changes. Instead of 

`25 megabytes within 100 seconds`

can you put:

`0.25 megabytes second`

And remove the trailing underscore. With those it'll be good to go!

Reply:
SORRY, make it `0.25 megabytes per second`

Reply:
@creativedrewy final version:
> **Warning**: on this step make sure your network is reliable and has a minimum upload speed of 0.25 megabytes per second.